### PR TITLE
Feature/nodes se stack

### PR DESCRIPTION
**Pull request by DezzK**

### CODE REVIEW

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 1]
```diff
@@ -1,112 +1,225 @@
...
             "previewUrl": ""
-            },
-            "type": "custom"
+          }
         },
         {
-            "commandLine": "cat | sh /home/user/bin/deploy_this.sh ${current.project.path} ${editor.current.file.name} ${editor.current.file.basename}",
-            "name": "Deploy Selected Contract", 
-            "attributes": {
+          "commandLine": "cat | sh /home/user/bin/deploy_this.sh ${current.project.path} ${editor.current.file.name} ${editor.current.file.basename}",
```
Comment (andrey-kurochkin) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: /home/user/bin/deploy_this.sh 
^
Can you double check if we want to keep absolute url?

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 2]
```diff
@@ -1,112 +1,225 @@
...
+                "org.eclipse.che.terminal",
+                "org.eclipse.che.ws-agent"
+              ],
+              "attributes": {
+                "memoryLimitBytes": "536870912"
+              }
+            },
+            "ws/se-node": {
+              "env": {},
+              "servers": {
```
Comment (andrey-kurochkin) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json:145]: I think we should add ADNL endpoint here too

**Review by andrey-kurochkin — CHANGES_REQUESTED** [review]

File: ide/che-core-ide-stacks/src/main/resources/stacks.json (hunk 2 above)
Comment (andrey-kurochkin) [reply ide/che-core-ide-stacks/src/main/resources/stacks.json:145]: Nevermind. I was mistaken. It won't work out of the box.

**Review by SeHor05 — COMMENTED** [review]
All deploy scripts will be in /opt/app/scripts/. Need to change PATH in commands.
All commands related to Node should be deleted. (Start, Stop, Restart Local Node)

**Review by andrey-kurochkin — APPROVED** [review]

**Pull request merged**